frontend/src/pages/home-page.tsx

```diff
@@ -52,8 +52,8 @@ function HomePage() {
       </div>
       <div className="mx-4 md:mx-8 lg:mx-16">
         <BlogFeed />
-        <h1 className="text-2xl font-semibold dark:text-dark-primary">All Posts</h1>
-        <div className="flex flex-wrap">
+        <h1 className="mb-4 text-2xl font-semibold dark:text-dark-primary">All Posts</h1>
+        <div className="flex flex-wrap gap-4">
           {posts.length === 0
```

Comment:
Why gap-4 is added? It is breaking the UI by pushing the posts to below at the right we have empty white space now
<img width="1549" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/75a750f1-008f-4e68-bbde-c85a22e5a049">

And not exactly sure why mb-4 is added here, i understand to compensate the m-4 removal in the post-card-skeleton, but then there would be other places also to be considered